e2e/_suites/fleet/fleet.go

```diff
@@ -1654,23 +1654,46 @@ func (fts *FleetTestSuite) thePolicyIsUpdatedToHaveSystemSet(name string, set st
 		}).Warn("We only support system system/metrics, log, logfile and linux/metrics policy to be updated")
 		return godog.ErrPending
 	}
+
+	var err error
+	var packageDS kibana.PackageDataStream
+	var kibanaInputs []kibana.Input
 	var metrics = ""
-	var file = ""
+
 	if name == "linux/metrics" {
-		file = "/linux_metrics.json"
 		metrics = "linux"
+		packageDS, err = fts.kibanaClient.GetIntegrationFromAgentPolicy(fts.currentContext, metrics, fts.Policy)
```

Comment:
For safety, linux metrics keeps the old behaviour and reading the metrics inputs from the "linux_metrics" golden file. @IgorGuz2000 I think it could follow the same approach, although I do not see any scenario using the "linux/metrics" scenario.

Reply:
Yes there is only one scenario there to verify the metrics exist

Reply:
But with current project layout, I cannot see it anymore. Do you have any clue on where it is?

Reply:
@IgorGuz2000 I'm going to merge this one, we can revisit the "linux_metrics" in follow ups. Thanks!